src/layerPart.cpp

```diff
@@ -95,7 +95,7 @@ void createLayerParts(SliceMeshStorage& mesh, Slicer* slicer)
 
     // OpenMP compatibility fix for GCC <= 8 and GCC >= 9
     // See https://www.gnu.org/software/gcc/gcc-9/porting_to.html, section "OpenMP data sharing"
-#if defined(__GNUC__) && __GNUC__ <= 8
```

Comment:
If `__GNUC__` is defined, it should never also have `__clang__` defined, right?

Reply:
Maybe, but Clang 10 with MinGW traps here.